qcodes_contrib_drivers/drivers/Attocube/ANC350.py

```diff
@@ -0,0 +1,510 @@
+﻿from qcodes import Instrument, Parameter
+from qcodes.instrument.channel import InstrumentChannel, ChannelList
+from qcodes.utils.validators import Numbers
+from qcodes_contrib_drivers.drivers.Attocube.ANC350Lib import ANC350LibActuatorType, ANC350v3Lib, ANC350v4Lib
+from typing import Any, Callable, Dict, Optional, Union
+
+
+class Anc350Axis(InstrumentChannel):
+    """
+    Representation of an axis of the ANC350
+
+    The Attocube ANC350 has 3 axis, one for every direction.
+
+    Args:
+        parent: the Instrument that the channel is attached to
+        name: the name of the axis itself
+        axis: the index of the axis (0..2)
+
+    Attributes:
+        position: Get the current position on a single axis
+        frequency: Set the frequency of the output signal. Depending on positioner type and usage of other axes one can
+            adjust the frequency from 1Hz up to 5kHz (only on one axis at one time is a frequency above 2kHz allowed)
+        amplitude: Value for the drive voltage of the piezo drive. Bychanging this value, the step size of the
+            positioner can be varied. Value for the drive voltage of the piezo drive. Bychanging this value, the step
+            size of the positioner can be varied.
+        status:
+        voltage: Sets the DC level on the voltage output when no sawtooth based motion and no feedback loop
+            is active.
+        target_position: Sets the target position for automatic motion (start_auto_move). For linear type actuators the
+            position unit is m, for goniometers and rotators it is degree.
+        target_range: Defines the range around the target position where the target is considered to be reached.
+        actuator: Selects the actuator to be used for the axis from actuator presets.
+        actuator_name: Get the name of the currently selected actuator
+        capacitance: Performs a measurement of the capacitance of the piezo motor and returns the result. If no
+            motor is connected, the result will be 0.
+            The function doesn't return before the measurement is complete; this will take a few seconds of time.
+    """
+
+    def __init__(self, parent: "ANC350", name: str, axis: int):
+        super().__init__(parent, name)
+
+        self._axis = axis
+
+        self.add_parameter("position",
+                           label="Position",
+                           get_cmd=self._get_position,
+                           unit="mm or m°")
+
+        self.add_parameter("frequency",
+                           label="Frequency",
+                           get_cmd=self._get_frequency,
+                           set_cmd=self._set_frequency,
+                           vals=Numbers(),
```

Comment:
vals=Ints(1, 5000)
In the manual, it is stated that the Frequency can be between 1 Hz to 5 kHz for one axis and between 1 Hz to 2 kHz for all axis, with the resolution of 1 Hz.